jcl/src/jdk.jcmd/share/classes/openj9/tools/attach/diagnostics/tools/Jps.java

```diff
@@ -33,12 +33,28 @@
  * A tool for listing Java processes and their information.
  *
  */
+@SuppressWarnings("nls")
 public class Jps {
 
+	private static final String DESCRIPTION =
+			"jps: Print a list of Java processes and information about them%n%n";
+
+	private static final String HELP_TEXT =
+			"Usage:%n" +
+			"    jps [-h] [-J<vm option>] [-l] [-q] [-m] [-v]%n" +
+			"    jps [-h] [-J<vm option>] [-q] [-lmv]%n%n" +
```

Comment:
@alexey-anufriev this can stay as `[-lqmv]` but need a note that `-q` takes precedence over the other options.

Reply:
But `q` flag composed with any other flag will lead to `illegal argument`. And block `[-lqmv]` would mean that they can be used together, which is not true anymore.

Reply:
> But q flag composed with any other flag will lead to illegal argument

Probably there was confusion here, https://github.com/eclipse/openj9/pull/11052#issuecomment-721127641 (I agreed) suggested that `q` composed with other flags should not be restricted.

Reply:
Yes, it's good the way it is.

Reply:
> Yes, it's good the way it is.

@pshipton are you leaning toward to current way "restrict `q` flag with others`?

Reply:
The help output is as we requested. The restriction on combining -q with other flags isn't what we discussed before, but the code is already written. I don't think it's a big deal either way. I plan to merge it the way it is once the testing completes.